src/test/ui/const-generics/const-generic-array-wrapper.rs

```diff
@@ -3,11 +3,11 @@
 #![feature(const_generics)]
 //~^ WARN the feature `const_generics` is incomplete and may cause the compiler to crash
 
-struct Foo<T, const N: usize>([T; {N}]);
+struct Foo<T, const N: usize>([T; N]);
 
-impl<T, const N: usize> Foo<T, {N}> {
+impl<T, const N: usize> Foo<T, N> {
```

Comment:
This is the only use in this file that this PR allows, I'm surprised the others were written `{N}` (as they never needed to be)